sdk/src/utils/thumbnail.rs

```diff
@@ -25,6 +25,13 @@ const THUMBNAIL_JPEG_QUALITY: u8 = 80;
 /// returns Result (format, image_bits) if successful, otherwise Error
 #[cfg(feature = "file_io")]
 pub fn make_thumbnail(path: &std::path::Path) -> Result<(String, Vec<u8>)> {
+    #[cfg(all(feature = "sfnt", feature = "add_font_thumbnails"))]
```

Comment:
Same weird comment on function here too.. guess the other pre-dated this PR

Reply:
Yeah, I copy pasted I guess. Didn't realize it was here too. OK. I removed from the other file.